features/steps/test_post_endpoint.rb

```diff
@@ -8,8 +8,9 @@ class Spinach::Features::TestPostEndpoint < Spinach::FeatureSteps
   end
 
   step "I create an Item" do
+
     options = {
```

Comment:
I would change this to:
```ruby
options = {
  body: {
    name: "Foo",
    quality: 4,
    sellIn: 4,
    description: "FooDescription"
  },
  headers: @auth_headers
}
```

As a general rule of thumb, I think it's OK to have hashes / dictionaries with one key / value pair on a single line, but once they have two or more key / value pairs, you should do one line per key / value pair.